### PR TITLE
Add draft of ggplot2 internship post

### PR DESCRIPTION
**Pull request by karawoo**
Here's a draft of the post about my ggplot2 internship experience. If you have any suggestions I'm happy to make edits. I know the title is a little bland...

### CODE REVIEW

File: content/articles/2017-08-ggplot2-internship.Rmd [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Daily life as a ggplot2 intern
+
+My ggplot2 work consisted of a few different things: issue and pull request
+triage, bug fixes, and developing new features. The tidyverse gets a lot of new
+issues opened on GitHub, so to help keep things organized I tagged each ggplot2
+issue with labels describing the type of issue and which part of ggplot2 it
+related to (scales, themes, layers, etc.). In some cases I helped the authors
+create reprexes so that we could diagnose the problems and determine when they
+were fixed. I triaged new pull requests as well, merging anything that was very
+minor, helping authors conform to the tidyverse style guide, and requesting
```
Comment (jennybc) [inline content/articles/2017-08-ggplot2-internship.Rmd]: "very minor" seems vaguely belittling to someone, I can't tell if it's you or the PRer or both. Maybe reword along these lines: "straightforward", "a clear win".

File: content/articles/2017-08-ggplot2-internship.Rmd [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+shapes). As part of this project I improved support for datetime scales and
+ordered factors. I also added built-in support for [viridis color
+palettes](https://bids.github.io/colormap/), a move that has garnered me some
+significant Twitter popularity. `r emo::ji("sunglasses")`
+
+I created a new position function, `position_dodge2()`, for placing box plots,
+bars, and rectangles. This project started out as a fix for a bug where boxes of
+differing widths could not be dodged from one another using the existing
+`position_dodge()` function, but it quickly grew into a larger project that
+fixed not only the original issue, but three other issues as well.
```
Comment (jennybc) [inline content/articles/2017-08-ggplot2-internship.Rmd]: Do you want to link to any that? I can understand if the answer is "no". But if you're neutral to positive, I think it helps other people get a very concrete sense of what a bug is, what a bug fix looks like, and how different bugs end up have common causes/fixes.

File: content/articles/2017-08-ggplot2-internship.Rmd [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+![](https://media.giphy.com/media/8VrtCswiLDNnO/giphy.gif)
+
+The last big effort was a mostly behind-the-scenes refactor of the way text gets
+placed on plots. This was a fun and challenging project that required me to get
+familiar with the grid graphics system for the first time. ggplot2 users should
+notice only very minor changes as a result of this project, but internally we've
+really cleaned up the code, reduced duplication, and documented a number of
+internal functions to make them easier to work with in the future.
```
Comment (jennybc) [inline content/articles/2017-08-ggplot2-internship.Rmd]: Are there specific geoms affected by this?

**Comment by jennybc** [comment]
I agree it looks great overall! I left a few specific comments, none of which are earth-shaking.

File: content/articles/2017-08-ggplot2-internship.Rmd (hunk 3 above)
Comment (karawoo) [reply content/articles/2017-08-ggplot2-internship.Rmd]: It's tricky because one of the big challenges here was to _not_ change existing behavior much. The main thing that is ever so slightly different is facet strip labels, which I'll mention.

**Comment by hadley** [comment]
Lets plan to publish both on Friday - can you please set the date accordingly?

**Comment by jennybc** [comment]
I think you'll need to change the month in the file names too @karawoo

**Comment by karawoo** [comment]
I changed it to `09`, does it need to be something different?

**Comment by hadley** [comment]
LGTM

**Comment by hadley** [comment]
Thanks!
